interface/forms/procedure_order/new.php

```diff
@@ -332,77 +377,94 @@ function addProcLine() {
 // This is for callback by the find-code popup.
 // Appends to or erases the current list of related codes.
 function set_related(codetype, code, selector, codedesc) {
- var f = document.forms[0];
- var s = f[rcvarname].value;
- if (code) {
-  if (s.length > 0) s += ';';
-  s += codetype + ':' + code;
- } else {
-  s = '';
- }
- f[rcvarname].value = s;
+    var f = document.forms[0];
+    var s = f[rcvarname].value;
+    if (code) {
+        if (s.length > 0) s += ';';
+        s += codetype + ':' + code;
+    } else {
+        s = '';
+    }
+    f[rcvarname].value = s;
 }
 
 // This invokes the find-code popup.
 function sel_related(varname) {
- rcvarname = varname;
- // codetype is just to make things easier and avoid mistakes.
- // Might be nice to have a lab parameter for acceptable code types.
- // Also note the controlling script here runs from interface/patient_file/encounter/.
- dlgopen('find_code_dynamic.php?codetype=<?php echo attr(collect_codetypes("diagnosis", "csv")); ?>', '_blank', 985, 750, '', '<?php echo xla("Select Diagnosis Codes"); ?>');
+    rcvarname = varname;
+    // codetype is just to make things easier and avoid mistakes.
+    // Might be nice to have a lab parameter for acceptable code types.
+    // Also note the controlling script here runs from interface/patient_file/encounter/.
+    dlgopen('find_code_dynamic.php?codetype=ICD10', '_blank', 825, 750, '', '<?php echo xla("Select Diagnosis Codes"); ?>');
```

Comment:
You should not need to hardcode ICD10 here. the collect_codetypes will do this (and allow folks to use other diagnosis code types as needed, which are set in administration->Lists->Code Types